packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -56,7 +59,11 @@ export const PaymentConfirmation = ({
       data-testid="payment-confirmation"
     >
       <header>
-        <img src={circledCheckbox} alt="circled checkbox" />
+        <img
+          className="circled-check"
+          src={circledCheckbox}
+          alt="circled checkbox"
```

Comment:
Not your fault but this alt text isn't very useful and it isn't localized. What do you think? Maybe empty alt tag?

Reply:
Maybe leave it and file a follow-up issue to fix it?  I do agree that the alt text isn't very helpful.

Reply:
Just filed https://jira.mozilla.com/browse/FXA-3047